index.html

```diff
@@ -5,16 +5,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/grid.css"> 
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<div class="container">
```

Comment:
This is a good use of the container as it 'contains' (literally) your entire page.
However, you will notice that because of this, your page elements don't span to the edges of the screen. In a lot of cases, depending on the design, you'll want the background colour of your nav and background image of your hero (for example), to grow as wide as the screen, but for _the content_ (i.e. the text) to be contained.
To achieve this, instead of wrapping the whole thing in a container, you can use the container inside individual sections. For example for a hero you'd do something like this:
```
    <section class="my-full-width-background-class">
        <div class="container">
            <div class="row">
                <div class="col-12">
                    <h4>This is the hero!</h4>
                </div>
            </div>
        </div>
    </section>
```
Note how we set the background image in the outer parent, and then _inside_ of that we put the container, to container the text etc.